cypress/integration/core/ozon-content.spec.ts

```diff
@@ -369,4 +369,31 @@ describe("Ozon Content", () => {
       .invoke('removeAttr', 'interactive')
       .and('have.css', 'text-decoration-line', 'none');
   });
+
+  it('should render Figuur as dso-image-overlay', () => {
+    cy.visit('http://localhost:56106/iframe.html?id=ozon-content--figuur');
+
+    cy.get('dso-ozon-content')
+      .invoke('prop', 'content', `
+        <Figuur
+          eId="chp_13__subsec_13.7__art_13.72__table_o_1__img_o_1"
+          wId="gm1979_2__chp_13__subsec_13.7__art_13.72__table_o_1__img_o_1"
+        >
+          <Titel>Afbeelding Titel</Titel>
+          <Illustratie
+            naam="https://www.dso-toolkit.nl/master/docs/images/global.jpg"
+            alt="Afbeelding 1"
+          />
+          <Bijschrift>Bijschrift bij het figuur.</Bijschrift>
+          <Bron>Bron waaruit het figuur is overgenomen</Bron>
+        </Figuur>
+      `)
+      .get('dso-ozon-content')
+      .find('dso-image-overlay > img')
+      .should('have.attr', 'src', 'https://www.dso-toolkit.nl/master/docs/images/global.jpg')
+      .and('have.attr', 'alt', 'Afbeelding Titel')
+      .and('not.have.attr', 'height', 'width')
```

Comment:
Ik kan deze constructie niet terugvinden in de documentatie, weet je zeker dat je hier checkt dat "geen attribuut height en width"?

Met mijn beperkte Cypress kennis verwacht ik dat we testen dat het attribuut "height" geen value "width" moet hebben.. Volgens mij verandert `.not` niet ineens het gedrag van de assertion, maar draait ie de verwachting om.

Reply:
Dit gaat inderdaad niet.